package/Runtime/Plugins/iOS/NimbusManager.swift

```diff
@@ -0,0 +1,196 @@
+//
+//  NimbusManager.swift
+//
+//  Created by Bruno Bruggemann on 5/7/21.
+//  Copyright © 2021 Timehop. All rights reserved.
+//
+
+import Foundation
+import NimbusRenderStaticKit
+import NimbusRenderVideoKit
+import NimbusKit
+
+@objc public class NimbusManager: NSObject {
+    
+    @objc public static let shared = NimbusManager()
+    
+    private let kCallbackTarget = "IOSAdManager"
+    
+    private var nimbusAdManager: NimbusAdManager?
+    private var adController: AdController?
+    
+    private var adView: AdView?
+    
+    @objc public func initializeNimbusSDK(publisher: String,
+                                          apiKey: String,
+                                          enableSDKInTestMode: Bool,
+                                          logLevel: Int,
+                                          appName: String,
+                                          appDomain: String,
+                                          bundleId: String,
+                                          storeUrlString: String,
+                                          showMuteButton: Bool) {
```

Comment:
This is different Android. What is the intent with a show mute button bool?

Reply:
That is tricky. If I remember correctly each platform implemented it in a different way. @inderdhir and @JasonSznol any thoughts on this?

Reply:
The behavior of the Mute button should be on all video interstitial and rewarded ads. I have a a switch that allows he publisher to set the actual icon of the mute button as a static method on the BlockingAdRenderer but we don't provide a means for them to optionally show or hide the mute button as part of the initilzation and the iOS sdk should handle rendering the mute icon automatically as part of the work we did a few weeks ago.

Reply:
I just talked to Inder, and we can turn it ON or OFF when the function to show ad video interstitial or rewarded ads, but I'm not seeing where we are passing it from unity. Should it be true by default for now?

Reply:
I would default the volume button to not be there by default to begin with and we can change that behavior in another release if necessary. A muted ad does not count as a viewable impression when measured by a company like MOAT. Unity Ads for example flashes a volume button for a brief second before continuing to the video ad but the volume is defaulted to on and not configurable after the video starts.

Reply:
Ok, I will confirm that with Brandon. Thanks for the suggestion.